codewof/package.json

```diff
@@ -19,7 +19,6 @@
     "@babel/core": "7.4.4",
     "@babel/preset-env": "7.4.4",
     "browserify": "16.2.3",
-    "child_process": "1.0.2",
     "del": "4.1.1",
     "gulp": "4.0.2",
```

Comment:
I think the license for `gulp-error-handle` has been missed.

https://github.com/MhMadHamster/gulp-error-handle/blob/master/LICENSE